docs/root/_static/searchtools.js

```diff
@@ -129,7 +129,7 @@ const _displayItem = (item, searchTerms) => {
 };
 const _finishSearch = (resultCount) => {
   Search.stopPulse();
-  Search.title.innerText = _("Search Results");
```

Comment:
im surprised that `innerText` could allow XSS (as opposed to `innerHTML`) - also thinking this is probably not a vector as its being _set_ with explicit text - either way the change seems to work - so happy to land

note this is not the only instance of usage in this file - i didnt check if any other were being set with ~untrusted text